src/apps/loan-list/list/loan-list.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useEffect, useState } from "react";
+import dayjs from "dayjs";
+import localizedFormat from "dayjs/plugin/localizedFormat";
```

Comment:
Great dayjs looks like a nifty tool. I think it would be a good idea to mention when we introduce a global tool like that in Zulip. I probably forgot it myself so this is also a reminder for myself.

Reply:
Noted.